docs/requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy>=1.13
 scipy>=1.5.0
-netCDF4
 pyparsing>=1.5.7
+netCDF4
```

Comment:
Why is `netCDF4` listed here? I also notice that it is still present in `../requirements.txt`.

Reply:
`requirements.txt` was a missing delete. But docs/ is necessary for building the documentation. There I just use everything.